pkg/collector/corechecks/snmp/internal/checkconfig/profile.go

```diff
@@ -101,24 +126,23 @@ func getDefaultProfilesDefinitionFiles() (profileConfigMap, error) {
 func loadProfiles(pConfig profileConfigMap) (profileConfigMap, error) {
 	profiles := make(profileConfigMap, len(pConfig))
 
-	for name, profile := range pConfig {
-		if profile.DefinitionFile != "" {
-			profileDefinition, err := readProfileDefinition(profile.DefinitionFile)
+	for name, profConfig := range pConfig {
+		if profConfig.DefinitionFile != "" {
+			profDefinition, err := readProfileDefinition(profConfig.DefinitionFile)
 			if err != nil {
 				log.Warnf("failed to read profile definition `%s`: %s", name, err)
 				continue
 			}
 
-			err = recursivelyExpandBaseProfiles(profileDefinition, profileDefinition.Extends, []string{})
+			err = recursivelyExpandBaseProfiles(profDefinition, profDefinition.Extends, []string{})
 			if err != nil {
 				log.Warnf("failed to expand profile `%s`: %s", name, err)
 				continue
 			}
-			profile.Definition = *profileDefinition
-			profiles[name] = profile
-		} else {
-			profiles[name] = profile
+			profConfig.Definition = *profDefinition
+			profiles[name] = profConfig
```

Comment:
```suggestion
```
I don't think we need this line now that you've moved it outside of the if block, do we?

Reply:
Good catch!